src/module/actor/character/document.ts

```diff
@@ -1472,7 +1473,11 @@ class CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
                         ? configuredAmmo.uses.value
                         : configuredAmmo.quantity
                     : (configuredAmmo?.quantity ?? 0);
-                params.consumeAmmo ??= expend > 0;
+                if (!weapon.system.ammo?.builtIn) {
+                    params.consumeAmmo = false;
+                } else {
+                    params.consumeAmmo ??= expend > 0;
+                }
```

Comment:
Consume only if builtin?

Reply:
Likely a mistake I introduced when swapping built in to a variable, thanks for catching.